spark-k8s/Dockerfile

```diff
@@ -319,20 +316,19 @@ microdnf clean all
 rm -rf /var/cache/yum
 
 ln -s /usr/bin/python${PYTHON} /usr/bin/python
+chown -h ${STACKABLE_USER_UID}:0 /usr/bin/python
```

Comment:
I don't think this is necessary. System wide binaries usually belong to root.

Reply:
done https://github.com/stackabletech/docker-images/pull/1042/commits/e7b6ed1e6712fbba4c3772711bc9d4554c3dffb1